src/R/Sql/Impl/Publish/DacPackageServices.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.R.Components.Sql.Publish;
+
+namespace Microsoft.VisualStudio.R.Sql.Publish {
+    [Export(typeof(IDacPackageServices))]
```

Comment:
`GetExportedValueOfDefault` throws if there is more than one export, and doesn't throw if there are no exports. If export is found, it there is no difference - exceptions in export loading logic aren't handled.
But, if we export a provider, we don't need to export `IDacPackageServices` themselves - `DacPackageServicesProvider.GetDacPackageServices` can explicitly create an instance.